static/app/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding.tsx

```diff
@@ -144,11 +144,11 @@ export const getCrashReportJavaScriptInstallStep = params => [
 
 export function getCrashReportSDKInstallFirstStep(params) {
   const version =
-    params.sourcePackageRegistries.data['sentry.javascript.browser'].version;
+    params.sourcePackageRegistries.data?.['sentry.javascript.browser'].version ?? '';
```

Comment:
Is the script still useable if these values don't exist?

Reply:
the values will always exist, they just take a sec to load. this check is in place to make sure we don't prematurely access unloaded data. will update this code to reference the `isLoading` param to be more explicit